test/integration/integration_test.exs

```diff
@@ -49,13 +49,21 @@ defmodule KafkaEx.Integration.Test do
     assert consumer_group_update_interval == 30000
   end
 
-  test "create_worker provides a default consumer_group of 'kafka_ex'" do
```

Comment:
I think that's fine, but I think it should be a separate setting - see the complication that I point out above.  Maybe something like

```
config :kafka_ex,
  consumer_group: "kafka_ex",
  disable_consumer_group: false
```